Source/WebCore/page/scrolling/ScrollingTreeGestureState.cpp

```diff
@@ -93,11 +93,11 @@ void ScrollingTreeGestureState::nodeDidHandleEvent(ScrollingNodeID nodeID, const
         break;
     case PlatformWheelEventPhase::Began:
         m_activeNodeID = nodeID;
-        m_scrollingTree.handleWheelEventPhase(nodeID, event.momentumPhase());
+        m_scrollingTree.get()->handleWheelEventPhase(nodeID, event.momentumPhase());
         break;
     case PlatformWheelEventPhase::Ended:
         if (m_activeNodeID)
-            m_scrollingTree.handleWheelEventPhase(*m_activeNodeID, event.momentumPhase());
+            m_scrollingTree.get()->handleWheelEventPhase(*m_activeNodeID, event.momentumPhase());
```

Comment:
We should store this in a `Ref` before using it.

Reply:
`m_scrollingTree` is a weak reference, so it won't prevent the object from being destroyed. Don't we need a strong reference on the stack since `handleWheelEventPhase` and others are non-trivial functions?

Reply:
I see, I missed that